extension/snippets/dvc-yaml.code-snippets

```diff
@@ -38,6 +38,19 @@
     ],
     "description": "Foreach stage template for dvc.yaml pipeline"
   },
+  "Matrix Pipeline Stage Template": {
+    "prefix": "dvc-pipeline-matrix-stage",
+    "body": [
+      "  ${1:train}:",
+      "    matrix:",
+      "      model: [${2:cnn}, ${3:xgb}]",
```

Comment:
I would replace the entire line here instead of the components of the matrix. E.g. 

```
" ${2:model: [cnn, xgb]}"
```

Model isn't necessarily going to be in every matrix. Same applies for the `feature` line and the command within cmd.

Reply:
Update the snippets to replace entire lines!